packages/app/src/breadcrumbs/BreadcrumbsBar.tsx

```diff
@@ -79,8 +76,10 @@ function BreadcrumbsBar(props: Props) {
           label="Go full screen"
           onClick={() => {
             if (!document.fullscreenElement) {
-              void document.documentElement.requestFullscreen();
-            } else if (document.exitFullscreen) {
```

Comment:
Not sure why I was checking for the existence of `exitFullscreen` here. It is well supported.